Sources/ScipioKit/Runner.swift

```diff
@@ -162,17 +115,180 @@ public struct Runner {
+            case disabled
+            case project
+            case storage(any CacheStorage)
+        }
+        public enum PlatformSpecifier: Equatable {
+            case manifest
+            case specific(Set<Platform>)
+        }
+
+        public enum Platform: String, Hashable {
+            case iOS
+            case macOS
+            case macCatalyst
+            case tvOS
+            case watchOS
+        }
+
+        public var buildOptionsContainer: BuildOptionsContainer
+        public var cacheMode: CacheMode
+        public var overwrite: Bool
+        public var verbose: Bool
+
+        public init(
+            baseBuildOptions: BuildOptions = .init(),
+            buildOptionsMatrix: [String: TargetBuildOptions] = [:],
+            cacheMode: CacheMode = .project,
+            overwrite: Bool = false,
+            verbose: Bool = false
+        ) {
+            self.buildOptionsContainer = BuildOptionsContainer(
+                baseBuildOptions: baseBuildOptions,
+                buildOptionsMatrix: buildOptionsMatrix
+            )
+            self.cacheMode = cacheMode
+            self.overwrite = overwrite
+            self.verbose = verbose
+        }
+    }
+}
+
+extension Runner.Options.Platform {
+    fileprivate func extractSDK(isSimulatorSupported: Bool) -> Set<SDK> {
+        switch self {
+        case .macOS:
+            return [.macOS]
+        case .macCatalyst:
+            return [.macCatalyst]
+        case .iOS:
+            return isSimulatorSupported ? [.iOS, .iOSSimulator] : [.iOS]
+        case .tvOS:
+            return isSimulatorSupported ? [.tvOS, .tvOSSimulator] : [.tvOS]
+        case .watchOS:
+            return isSimulatorSupported ? [.watchOS, .watchOSSimulator] : [.watchOS]
+        }
+    }
+}
+
+extension Runner.Options.BuildOptions {
+    fileprivate func makeBuildOptions(descriptionPackage: DescriptionPackage) -> BuildOptions {
+        let sdks = detectSDKsToBuild(
+            platforms: platforms,
+            package: descriptionPackage,
+            isSimulatorSupported: isSimulatorSupported
+        )
+        return BuildOptions(
+            buildConfiguration: buildConfiguration,
+            isDebugSymbolsEmbedded: isDebugSymbolsEmbedded,
+            frameworkType: frameworkType,
+            sdks: OrderedSet(sdks)
+        )
+    }
+
+    private func detectSDKsToBuild(
+        platforms: Runner.Options.PlatformSpecifier,
+        package: DescriptionPackage,
+        isSimulatorSupported: Bool
+    ) -> Set<SDK> {
+        switch platforms {
+        case .manifest:
+            return Set(package.supportedSDKs.reduce([]) { sdks, sdk in
+                sdks + (isSimulatorSupported ? sdk.extractForSimulators() : [sdk])
+            })
+        case .specific(let platforms):
+            return Set(platforms.reduce([]) { sdks, sdk in
+                sdks + sdk.extractSDK(isSimulatorSupported: isSimulatorSupported)
+            })
+        }
+    }
```

Comment:
Could you explain the difference of manifest and specific?

Reply:
`manifest` is read from the manifest files of the package description.

`specific` is passed from CLI.

In create mode, if platforms are not specified by users, Scipio tries to build for platforms defined in `Package.swift` of the package.